00-open-a-shapefile.Rmd

```diff
@@ -220,15 +206,6 @@ extent(aoiBoundary)
 
 ```
```

Comment:
hey @jsta i added thos links because we introduce plot above and don't get into the various settings for colors, etc. It seems that having links to more about plotting parameters is very relevant here. We could create a note for it OR it could become additional reading at the top. But when i went through this lesson, i was thinking - how do i change the colors, etc. How did they know what to use in the plot parameters. I'd leave this as a note where they create the plot.